### PR TITLE
feat: update editor devtools

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

* Update develop tools for `apps/editor`.
  * Update `babel` & `karma` : https://github.com/nhn/tui.editor/pull/721/commits/ee900bd66e8f0fad32752a926d1b7a69c6a94ed0
  * ~Change default export : https://github.com/nhn/tui.editor/pull/721/commits/63c8df156fda913b8a823b5f70a347e28b30b523~
  * Apply babel to monorepo : https://github.com/nhn/tui.editor/pull/721/commits/76743eaeb65bf4916333c853d78298c5753e9036

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/src/js/extensions/editorProxy.js [hunk 1]
```diff
@@ -7,11 +7,11 @@
...
 try {
-  Editor = require('../editor');
+  Editor = require('../editor').default;
 } catch (e) {}
 if (!Editor) {
   try {
-    Editor = require('../viewer');
+    Editor = require('../viewer').default;
   } catch (e) {}
 }
```
Comment (dongwoo-kim) [inline apps/editor/src/js/extensions/editorProxy.js]: 이 파일은 일단 유지하는 건가요? 익스텐션 분리하면서 사라질 줄 알았는데..

Comment (seonim-ryu) [reply apps/editor/src/js/extensions/editorProxy.js]: @dongwoo-kim 
이건.. 이 브랜치 머지되면 사라질 예정입니다ㅎㅎ (여기선 익스텐션 제거하지 않거든요...)

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    presets: [
+      [
+        '@babel/preset-env',
+        {
+          modules: false,
+          loose: true
+        }
+      ]
+    ],
+    plugins: [['@babel/plugin-proposal-class-properties']],
```
Comment (seonim-ryu) [inline babel.config.js]: 브라켓 제거 완료

* https://github.com/nhn/tui.editor/pull/721/commits/fdca2a9ec1f4e97acccabed29ea78954584a0efc

**Comment by seonim-ryu** [comment]
@todo `indexAll.js`, `indexViewerAll.js` 파일 유지 여부는 **번들 파일 개선** 피처에서 처리

**Pull request merged**